site/docs/components/accordion/examples.mdx

```diff
@@ -36,6 +36,28 @@ You can use `Accordion`'s controlled API to allow only one accordion to be expan
 
 Use the `disabled` prop to disable an accordion. The user can't expand, collapse or focus a disabled accordion.
 
+</LivePreview>
+<LivePreview componentName="accordion" exampleName="IndicatorSide" >
+
+## Indicator side
```

Comment:
Any need to mention content indentation?

Reply:
@bhoppers2008 fyi, added :
```_Note that the content left aligns to the text in accordion header, so moving the indicator to the right side will also change the content's left padding._```